examples/notebooks/rag/rag_1A_dpk_process_ray.ipynb

```diff
@@ -303,7 +303,8 @@
     "    \"data_files_to_use\": ast.literal_eval(\"['.pdf']\"),\n",
     "    # orchestrator\n",
     "    \"runtime_worker_options\": ParamsUtils.convert_to_ast(worker_options),\n",
-    "    \"runtime_num_workers\": MY_CONFIG.RAY_RUNTIME_WORKERS,\n",
+    "    # \"runtime_num_workers\": MY_CONFIG.RAY_RUNTIME_WORKERS,\n",
```

Comment:
Why is this commented out. Please remove or go back to using MY._CONFIG.RAY_RuNTIME_WORKERS = 1

Reply:
Setting `MY._CONFIG.RAY_RUNTIME_WORKERS = 1` breaks some other transforms: ededupe and fuzzy-dedupe.
They seem to need at least 2 workers.

So I am setting workers = 1 only for PDF2PQ step (so we don't run into the race condition of cleaning out the downloaded models.

This is a temp fix until other worker issues are resolved.